types/react-native-easy-upgrade/index.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for react-native-easy-upgrade 0.0
+// Project: https://github.com/react-native-org/react-native-easy-upgrade#readme
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+interface Options {
+	iOSAppId: string;
+	iOSAppLookupUrl: string;
+	downloadTitle: string;
+	downloadDescription: string;
+	downloadDestDirectory: string;
+	downloadApkName: string;
+	downloadApkEnd(path: string): unknown;
+	shouldCheckApkHasDownloaded: boolean;
+	onError(err: Error): unknown;
+}
+
+declare class RNEasyUpgrade {
```

Comment:
This _appears_ to be named incorrectly, according to https://unpkg.io/react-native-easy-upgrade@0.0.9/index.js. The structure looks right but the name is `AppUpgrade`. `RNEasyUpgrade` appears to be something else, although I don‘t entirely understand what.